ReleaseProcessAutomation/Steps/PipelineSteps/ReleaseAlphaBetaStep.cs

```diff
@@ -95,13 +96,14 @@ public void Execute (SemanticVersion nextVersion, string? commitHash, bool pause
 
     var nextJiraVersion = InputReader.ReadVersionChoice("Please choose next version (open JIRA issues get moved there):", nextPossibleJiraVersions);
 
-    _releaseVersionAndMoveIssuesSubStep.Execute(nextVersion, nextJiraVersion);
+    if (!_releaseVersionAndMoveIssuesSubStep.Execute(nextVersion, nextJiraVersion, squashUnreleased: false, movePreReleaseIssues: true))
+      return;
 
     _msBuildCallAndCommit.CallMSBuildStepsAndCommit(MSBuildMode.PrepareNextVersion, nextVersion);
 
     if (pauseForCommit)
       return;
 
-    _continueAlphaBetaStep.Execute(nextVersion, currentBranchName, currentBranchName, noPush);
+    _continueAlphaBetaStep.Execute(nextVersion, ancestor: currentBranchName, currentBranchName, noPush);
```

Comment:
I would reset this change or do you think the annotation adds any value? If it does we should also include it at the other parameters since their intention is also unclear without an annotation IMHO

```suggestion
    _continueAlphaBetaStep.Execute(nextVersion, currentBranchName, currentBranchName, noPush);
```